server/service/devices.go

```diff
@@ -24,6 +24,7 @@ type getDeviceHostResponse struct {
 	Host       *HostDetailResponse `json:"host"`
 	OrgLogoURL string              `json:"org_logo_url"`
 	Err        error               `json:"error,omitempty"`
+	License    fleet.LicenseInfo   `json:"license"`
```

Comment:
Should device users have access to `device_count` and `expiration`? (If not, we could define and expose a custom `LicenseInfoLite` type)

Reply:
/cc @noahtalerman @chiiph

Reply:
@lukeheath I was thinking something similar although I know we were under time constraints so I went with what was spec'd. How do you feel about including this change in my revisions now?

Reply:
@gillespi314 This revision makes sense to me. Thanks for iterating so I could get the fix in! 🙇